website/static/js/components/legend.js

```diff
@@ -1,17 +1,56 @@
 require('css/legend.css');
 var m = require('mithril');
 
+/** Finds the index of a string in an array
+ *
+ * @param {Array} array The array to search through
+ * @param {String} str The string to find
+ * @returns {*}
+ */
+var find_index = function (array, str) {
+    for (var i = 0; i < array.length; i++) {
+        if (array[i].label === str)
+            return i;
+    }
+    return -1;
+};
+
+/** Removes an index from an array immutably
+ *
+ * @param {Array} array the array to modify
+ * @param {Int} index the index to remove
+ * @returns {*} a new array missing the index value
+ */
+var remove_index = function (array, index) {
+    var new_array = [];
+    for (var i = 0; i < array.length; i++) {
+        if (i !== index)
+            new_array.push(array[i]);
+    }
+    return new_array;
+};
+
 module.exports = {
-    view: function(data, repr, opts) {
-        if(data[0].label){
-            data.sort(function(a, b) {
+    view: function (data, repr, opts) {
+        var other_index = find_index(data, 'Other');
```

Comment:
Ah okay, I thought the issue called for the ordering to be as such:

![screen shot 2015-12-18 at 10 58 10 am 2](https://cloud.githubusercontent.com/assets/8094255/11900877/5ab1f1fe-a576-11e5-9ff9-958a6869746a.png)